int8 id
float32 angle_ref
float32 rpm_ref
float32 current_ref